### PR TITLE
update webpack config to work with current dependencies

### PR DESCRIPTION
**Pull request by MarcoGlauser**
Hi there,
Thank you for creating this starter theme!
With the current dependencies, the webpack configuration won't work out of the box. 
Mainly the CopyWebpackPlugin changed the configuration syntax.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -35,16 +35,15 @@ if ( isProduction ) {
...
-						}
-					},
-					extensions: ['php', 'js', 'svg', 'css',]
-				}
-			],
+			defaultExtractor: content => {
+    				// Capture as liberally as possible, including things like `h-(screen-1.5)`
+    				const broadMatches = content.match(/[^<>"'`\s]*[^<>"'`\s:]/g) || []
+
+    				// Capture classes within other delimiters like .block(class="w-1/2") in Pug
```
Comment (cjkoepke) [inline webpack.config.js:42]: Is this newer syntax for Tailwind? I don't remember class names like this. Please include a link to some docs. Thanks! This also includes the above option.

File: webpack.config.js [hunk 2]
```diff
@@ -113,13 +111,12 @@ const config = {
...
-			from: './assets/images/',
-			to: 'images',
-			ignore: [
-				'.DS_Store'
-			]
-		}]),
+		new CopyWebpackPlugin({
+			patterns: [{
+                        	from: './assets/images/',
+                        	to: 'images',
```
Comment (cjkoepke) [inline webpack.config.js:117]: include the .DS_Store ignore (unless that's default now).

**Review by cjkoepke — CHANGES_REQUESTED** [review]

**Comment by cjkoepke** [comment]
@MarcoGlauser Thanks for contributing! Left some questions before I merge.

File: webpack.config.js (hunk 1 above)
Comment (MarcoGlauser) [reply webpack.config.js:42]: purgecss changed the way they define extractors.That's why I decided to copy the ones that tailwind recommends in their docs: https://tailwindcss.com/docs/controlling-file-size#setting-up-purge-css-manually

File: webpack.config.js (hunk 2 above)
Comment (MarcoGlauser) [reply webpack.config.js:117]: I'll look into this in the next few days. they removed the top level ignore but there's some replacement.

**Comment by saifulazfar** [comment]
may u guys check for this as well..i got these error too

> ERROR in ./assets/css/app.css
> Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
> ModuleBuildError: Module build failed (from ./node_modules/css-loader/dist/cjs.js):
> ValidationError: Invalid options object. CSS Loader has been initialized using an options object that does not match the API schema.
>  - options has an unknown property 'context'. These properties are valid:

**Comment by MarcoGlauser** [comment]
@cjkoepke Thanks for the feedback! I just added the requested changes. 

@saifulazfar That error should be fixed by this change https://github.com/cjkoepke/wp-tailwind/pull/11/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dL90

**Review by cjkoepke — APPROVED** [review]

**Pull request merged**